rust/agents/scraper/src/db/block.rs

```diff
@@ -1,90 +1,90 @@
-//! SeaORM Entity. Generated by sea-orm-codegen 0.9.3
+use std::hash::{Hash, Hasher};
 
-use sea_orm::entity::prelude::*;
+use ethers::prelude::H256;
+use eyre::{Context, Result};
+use sea_orm::{
+    prelude::*, ActiveValue::*, DbErr, EntityTrait, FromQueryResult, Insert, QueryResult,
+    QuerySelect,
+};
+use tracing::trace;
 
-#[derive(Copy, Clone, Default, Debug, DeriveEntity)]
-pub struct Entity;
+use abacus_core::BlockInfo;
 
-impl EntityName for Entity {
-    fn table_name(&self) -> &str {
-        "block"
-    }
-}
+use crate::conversions::{format_h256, parse_h256};
+use crate::date_time;
+use crate::db::ScraperDb;
 
-#[derive(Clone, Debug, PartialEq, DeriveModel, DeriveActiveModel)]
-pub struct Model {
+use super::generated::block;
+
+/// A stripped down block model.
```

Comment:
Took me a bit to realize this struct was used for, may be worth a note. It didn't click till I read chain_scraper/mod.rs

Reply:
Improved doc